### DISCUSSION TITLE
Copy button does not work for me

### DISCUSSION THREAD

**Discussion by FrostyColdMug** [CorentinTh/it-tools#349] [Q&A] [answered]
When I click on "Copy xxx" I get a pop up saying "xxx copied to clipboard" but when I try to paste it I get what I had in there previous.

Any suggestions on troubleshooting this?

Thanks

**Comment by cgoIT** [comment, marked as answer]
Maybe duplicate of #342

**Comment by CorentinTh** [comment]
I'll trigger a warning and prevent the pop up if copying is not availlable
